lib/ExpireShare.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace OCA\Testing;
+
+use DateTime;
+use OC\OCS\Result;
+use OCP\IRequest;
+use OCP\Share\Exceptions\GenericShareException;
+use OCP\Share\Exceptions\ShareNotFound;
+use OCP\Share\IManager;
+use OCP\Share\IShare;
+
+class ExpireShare {
+
+	/**
+	 * @var IRequest
+	 */
+	private $request;
+
+
+	/** @var IManager */
+	private $shareManager;
+
+	/**
+	 * @param IRequest $request
+	 * @param IManager $shareManager
+	 */
+	public function __construct(
+		IRequest $request,
+		IManager $shareManager
+	) {
+		$this->request = $request;
+		$this->shareManager = $shareManager;
+	}
+
+	/**
+	 * @param $id
+	 * @return IShare
+	 * @throws ShareNotFound
+	 */
+	private function getShareById($id) {
+		$share = null;
+
+		// First check if it is an internal share.
```

Comment:
```suggestion
		// check if it is an internal share.
```
Either add the comment for the next step as well, or remove the word `first`